spec/models/resource_action_serializer_spec.rb

```diff
@@ -4,14 +4,16 @@
   describe "#serialize" do
     let(:resource_action) do
       ResourceAction.new(
-        :dialog_id     => 123,
-        :resource_id   => 321,
-        :created_at    => Time.now,
-        :updated_at    => Time.now,
-        :resource_type => "DialogField",
-        :ae_namespace  => "Customer/Sample",
-        :ae_class      => "Methods",
-        :ae_instance   => "Testing"
+        :dialog_id                   => 123,
+        :resource_id                 => 321,
+        :created_at                  => Time.now,
+        :updated_at                  => Time.now,
```

Comment:
You never have to hardcode created_at and updated_at.  In particular, I'm not sure this is even right because our database uses UTC, so even with hardcoding/mocking, this should be Time.now.utc.